src/icon2svg.rs

```diff
@@ -0,0 +1,163 @@
+//! Produces svgs of icons in Google-style icon fonts
+
+use crate::{error::DrawSvgError, iconid::IconIdentifier};
+use kurbo::{Affine, BezPath, PathEl, Point};
+use skrifa::{
+    instance::{LocationRef, Size},
+    outline::DrawSettings,
+    raw::TableProvider,
+    FontRef, MetadataProvider,
+};
+use write_fonts::pens::{BezPathPen, TransformPen};
+
+fn round2(v: f64) -> f64 {
+    (v * 100.0).round() / 100.0
+}
+
+fn push_point(svg: &mut String, prefix: char, p: Point) {
+    svg.push(prefix);
+    svg.push_str(&format!("{},{}", round2(p.x), round2(p.y)));
```

Comment:
in general it's probably better to use the `write!` macro directly, instead of `format!`? format! allocates a new buffer and then copies the contents, but `write!` can write directly into the provided buffer.

Reply:
TIL. Or possibly relearned. That would be a good clippy. Will fix.